dockers/Screenshotter/screenshotter.js

```diff
@@ -393,7 +440,8 @@ function takeScreenshot(key) {
              * output file name for one of these cases, we accept both.
              */
             key += "_alt";
-            file = path.join(dstDir, key + "-" + opts.browser + ".png");
+            basename = key + basename.slice(key.length - 4);
```

Comment:
Actually I think I my thought was that we might have to handle other screenshots in a similar way one day. So `key.length - 4` is the length of the key before I added the `"_alt"` to it. The slice is everything after that, i.e. all the things that got appended to the original key. I'll try an alternate formulation in my next commit.